thirdparty/CMakeLists.txt

```diff
@@ -7,24 +11,25 @@ set(YAML_CPP_BUILD_CONTRIB OFF CACHE BOOL "")
 set(YAML_BUILD_SHARED_LIBS OFF CACHE BOOL "")
 set(YAML_CPP_INSTALL OFF CACHE BOOL "")
 set(YAML_CPP_FORMAT_SOURCE OFF CACHE BOOL "")
-FetchContent_Declare(
-    yaml-cpp
-    GIT_REPOSITORY      https://github.com/jbeder/yaml-cpp
-    GIT_TAG             1b50109f7bea60bd382d8ea7befce3d2bd67da5f
-)
-
-FetchContent_MakeAvailable(yaml-cpp)
-loco_thirdparty_target_compile_link_flags(yaml-cpp)
 
-## span-lite
-FetchContent_Declare(
-    span-lite
-    GIT_REPOSITORY      https://github.com/martinmoene/span-lite
-    GIT_TAG             v0.10.3
-    GIT_SHALLOW         ON
-)
+IF ( NOT yaml-cpp_FOUND AND NOT span-lite_FOUND ) 
```

Comment:
One final comment and i think we are ready to merge. Could you make the cmake lower case for function like calls such as `IF` and `ENDIF`. Add a new line to the end of the file. Move L5-L13 to within the `if` as its just for fetchcontent.